configuration.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+from selenium import webdriver
+import os
+import tempfile
+
+URL = "https://www.epam.com/"
+URL_CONTACT_US = "https://www.epam.com/about/who-we-are/contact"
+URL_ABOUT = "https://www.epam.com/about"
+BASE_URL = "https://petstore.swagger.io/v2"
+
+
+@pytest.fixture(params=["chrome", "firefox"], scope="function")
```

Comment:
It can be done with @pytest.mark.parametrize

Reply:
I prefer to use Fixture in this case, because it will close browser automatically